mirgecom/initializers.py

```diff
@@ -462,6 +450,218 @@ def exact_rhs(self, discr, q, t=0.0):
         return flat_obj_array(massrhs, energyrhs, momrhs)
 
 
+class MulticomponentLump:
+    r"""Solution initializer for multi-component N-dimensional Gaussian lump of mass.
+
+    The Gaussian lump is defined by:
+
+    .. math::
+
+         \rho = 1.0\\
+         {\rho}\vec{V} = {\rho}\vec{V_0}\\
+         {\rho}E = (\frac{p_0}{(\gamma - 1)} + \frac{1}{2}\rho{|V_0|}^{2}
+         {\rho~Y_\alpha} = {\rho~Y_\alpha}_{0}
+         + {\rho~Y_\alpha}_{a_\alpha}{e}^{(1-{r_\alpha}^{2})},
```

Comment:
`{\rho~Y_\alpha}_{a_\alpha}` is kind of awkward. How about just `{\rho~a_\alpha}`? (And then `a` instead of `{\rho}_{a}` in `Lump` for symmetry?)

Also maybe try using `&` before all the `=`'s to vertically align the equations? Looks a little nicer IMO.

Reply:
de488a8